test/UnsafeTest/src_90/org/openj9/test/unsafe/UnsafeTestBase.java

```diff
@@ -415,11 +546,52 @@ protected void testByte(Object target, String method) throws Exception {
 					+ ", Data: " + modelByte[i] + ", Index: " + i);
 			if (method.equals(VOLATILE)) {
 				myUnsafe.putByteVolatile(base(target, i), offset, modelByte[i]);
-			}
-			 if (method.equals(ORDERED)) {
-				myUnsafe.putObjectRelease(base(target, i), offset, new Byte(
-						(byte) -1));
-			} else {
+
+			} else if (method.equals(OPAQUE)) {
+				myUnsafe.putByteOpaque(base(target, i), offset, modelByte[i]);
```

Comment:
Why do these first few branches of the if not have a `checkTrueAt` call?

Reply:
`checkTrueAt` is to verify the return value. the `puts` that the first few branches are testing have void return types.

Reply:
Thanks for clarifying - I had confused the checkTrueAt with the assert on the written value which occurs in the caller.